lib/numbers.md

```diff
@@ -5,20 +5,21 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+In Ruby, the two main kinds of numbers are fixnum, which is an integer, or whole number without a decimal, and float, which is a number with a decimal.
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
+Common operations used are +, -, *, /, ** for addition, subtraction, multiplication, division and to the power. You can also compare numbers using <, >, <=, >= for less than, greater than, less than or equal to and greater than or equal to.
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+The "+" operation on a number will perform a the mathematical operation of addition. Example: 5 + 5 = 10
+The "+" operation on a string will combine strings. Example: "dog" + "3" = "dog3"
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
-Replace me with your response...
+No, it will not work because a string does not recognize the symbols as mathematical operations. The symbols for multiplication and division can serve different functions in Ruby beyond the mathematical operations.
```

Comment:
Did you TIAS (Try It And See) with `"20"` and, let's say `3`? What happens when you run `"20" / 3` and `"20" * 3` ?

If one or more of those doesn't work, **how** would you make it work?